Added linux as build target

hunting4code opened the pull request:
This is the result of running `flutter create --platforms=linux .` This command adds some changes to the pubspec.lock file but I was able to revert those changes with no issue. It also add a unit test file unter /test which I also removed.

Otherwise, this commit consists of just running the above command.

hunting4code committed:
Added linux as build target

This is the result of running `flutter create --platforms=linux .`
This command adds some changes to the pubspec.lock file but I was able
to revert those changes with no issue. It also add a unit test file
unter /test which I also removed.

Otherwise, this commit constists of just running the above command.

clragon reviewed (changes requested):
RE: Test files
In other projects I sometimes create a marker file in the place where flutter generates its example test file, to prevent it from continuously regenerating it. 

Generally, it would be nicer if flutter would make it either to regenerate a folder, but alas. Maybe we can open an issue for https://github.com/flutter/flutter/tree/main/packages/flutter_tools at some point.

hunting4code committed:
Removed linux platform from .metadata file

This is to prevent the file from being regenerated and it appears to be
useless.
